yarn.lock

```diff
@@ -6500,7 +6500,7 @@ mkdirp@^1.0.3:
   resolved "https://registry.yarnpkg.com/mkdirp/-/mkdirp-1.0.4.tgz#3eb5ed62622756d79a5f0e2a221dfebad75c2f7e"
   integrity sha512-vVqVZQyf3WLx2Shd0qJ9xuvqgAyKPLAiqITEtqW0oIUjzo3PePDd6fW9iFz30ef7Ysp/oiWqbhszeGWW2T6Gzw==
 
-moment@^2.24.0, moment@^2.25.3, moment@~2.29.1:
+moment@^2.24.0, moment@^2.25.3:
```

Comment:
Can't we actually remove moment now or is it still needed for something?

Reply:
That is here in the lockfile because its a hard dependency of antd during install (in its package.json). I've compiled the bundles though and saw that moment is actually not there anymore, so it is not imported by antd at runtime :+1:

Reply:
The little bit sobering aspect is that with switching from moment.js to date-fns, the bundle size changed from 3.30 MB to 3.28 MB (and we haven't even imported the UTC-relevant package from date-fns yet :-(

Reply:
In this case, is it then worth going through these changes and risk introducing bugs, or better cut our losses and drop this change?

Reply:
I don't know where you looked for this bundle size. I was looking at what it loads in the production build where the optimisations are applied.

@adrianaluchian: I wouldn't throw in the towel just yet.

Reply:
I compared the size of the js files in dist for the production build, but as I measured only with the file explorer I don't have more accurate numbers. I agree, we should not throw in the towel, small changes combined will also have a huge impact, I was just surprised that it is not that much, actually.

Reply:
But isn't what counts what gets delivered to the browser? When I get to the website, I have to load for instance the "view-bundle", that one is what counts or? On the server you also have stuff that only has to do with the server side, or am I missing something?

Reply:
OK, these are the "official" numbers, taken directly from the bundler output:

Main branch:
~~~
[10:07:02] Starting 'buildTestAppJs'...
test-app/dist/edit-bundle.js: 457.30 kB
test-app/dist/settings-bundle.js: 40.74 kB
test-app/dist/chunk-RCM2QDPR.js: 1.94 kB
test-app/dist/view-bundle.js: 14.46 kB
test-app/dist/chunk-ZQIMAAA7.js: 674.46 kB
test-app/dist/chunk-M7TF3I6H.js: 1681.64 kB
test-app/dist/midi-53MIUYGM.js: 428.45 kB
test-app/dist/chunk-FJBKNBIP.js: 2.01 kB
TOTAL: 3301.00 kB
~~~

This branch:
~~~
[10:07:52] Starting 'buildTestAppJs'...
test-app/dist/edit-bundle.js: 457.21 kB
test-app/dist/settings-bundle.js: 40.74 kB
test-app/dist/chunk-J2XJBGNO.js: 1.94 kB
test-app/dist/view-bundle.js: 14.35 kB
test-app/dist/chunk-QDZTRKRA.js: 674.46 kB
test-app/dist/chunk-OV5JMYNK.js: 1661.49 kB
test-app/dist/midi-53MIUYGM.js: 428.45 kB
test-app/dist/chunk-FJBKNBIP.js: 2.01 kB
TOTAL: 3280.64 kB
~~~

Reply:
So, my explorer was pretty accurate. Anyways, it's 20 kB less and that's good, we just have to continue optimizing IMO.